README.md

```diff
@@ -7,27 +7,29 @@ BitmovinConvivaAnalytics is available through [CocoaPods](https://cocoapods.org)
 To install it, simply add the following line to your Podfile:
 
 ```ruby
-pod 'BitmovinConvivaAnalytics', git: 'https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva.git', tag: '1.4.0'
+pod 'BitmovinConvivaAnalytics', git: 'https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva.git', tag: '2.0.0'
 ```
 
 Then, in your command line run:
 
 ```
+pod repo update
 pod install
 ```
 
 ### Compatibility
 
-The most recent version of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 2.57.1`.
-For `BitmovinPlayer` version `>= 2.57.1` please use [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.
+The 2.x versions of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 3.0.0`.
+
+For `BitmovinPlayer` version `>= 2.57.1` and `< 3.0.0` please use    [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.
```

Comment:
Nit: Should state Conviva integration version `>= 1.3.0 && < 2.0.0` IMO. It's implicit but might be worth making it explicit.

Reply:
Done